robot/Cumulus/tests/browser/recurring_donations/create_recurring_donation_and_check_rollups.robot

```diff
@@ -0,0 +1,99 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Create Recurring Donation And Check Rollups
+    [Documentation]              This test verifies that Account, Contact, and Soft Credit Contact rollups are udpated
+    ...                          when a Recurring Donation's Opportunities are Closed Won.
+    
+    #Create a Recurring Donation from a Contact
+    ${ns} =  Get NPSP Namespace Prefix
+    &{contact} =                 API Create Contact           Email=jjoseph@robot.com
+    Go To Record Home            &{contact}[Id]
+    &{recurringdonation} =       API Create Recurring Donation  npe03__Contact__c=&{contact}[Id]
+    ...                          Name=Julian Recurring Donation
+    ...                          npe03__Amount__c=100
+    ...                          npe03__Installments__c=12
+    ...                          npe03__Installment_Period__c=Monthly
+    Go To Record Home            &{recurringdonation}[Id]
+
+    #Find 1st Opportunity for Recurring Donation and Close It
```

Comment:
@Julian88Tex  you can create this keyword in NPSP.robot file 
API Query
    [Arguments]      ${table}    ${select}    &{fields}
    @{object} =         Salesforce Query             ${table}
    ...                 select=${select}
    ...                 &{fields}
    [return]            @{object}

to use in test case file 
@{opportunity1}=    API Query    Opportunity    Id   
    ...                          npe03__Recurring_Donation__c=&{recurringdonation}[Id]
    ...                          ${ns}Recurring_Donation_Installment_Name__c=(1 of 12)

sorry spacing might be messed up here

Reply:
@skristem hm, a bit confused what the purpose would be of creating a keyword vs. just using Salesforce Query. Is there something I'm missing? Seems like Keyword would almost exactly the same thing and just reduce it by one line.

Reply:
hmm actually it would make more sense if I was already mentioning the table and fields in the keyword and that way if we ever had to change the table name and fields u change it one place and be done instead of touching all the files. I tried to make it more generic and didn't think through. If you want to leave as it is thats fine or want to change then use 
API Query Opportunity
[Arguments] ${id} ${installment} &{fields}
@{object} = Salesforce Query Opportunity
... select=Id
... npe03__Recurring_Donation__c=${id}
... ${ns}Recurring_Donation_Installment_Name__c=${installment}
... &{fields}
[return] @{object}
and in test case use
@{opportunity1}= API Query Opportunity    &{recurringdonation}[Id]   (1 of 12)

Reply:
@skristem made this change and would appreciate you taking one final look. Thanks!

Reply:
looks good @Julian88Tex

Reply:
@skristem thanks!